tests/simpleharness/simpleharness.csproj

```diff
@@ -6,7 +6,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
I would like to print "warning" message (from the API), so users will know that this metric will not be captured.